hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/HddsServerUtil.java

```diff
@@ -551,15 +545,11 @@ public static void writeDBCheckpointToStream(DBCheckpoint checkpoint,
           }
         }
       }
-    } catch (CompressorException e) {
-      throw new IOException(
-          "Can't compress the checkpoint: " +
-              checkpoint.getCheckpointLocation(), e);
     }
   }
 
-  private static void includeFile(File file, String entryName,
-      ArchiveOutputStream archiveOutputStream)
+  public static void includeFile(File file, String entryName,
```

Comment:
If DBCheckPointServlet.writeDBCheckpointToStream() is removed, this method should stay private.